lib/plugins/boluswizardpreview.js

```diff
@@ -121,7 +119,7 @@ function init() {
   bwp.updateVisualisation = function updateVisualisation (sbx) {
 
     var results = sbx.properties.bwp;
-    if (results == undefined) return;
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Expected '===' and instead saw '=='.](https://www.codacy.com/app/Nightscout/cgm-remote-monitor/file/761367804/issues/source?bid=2452379&fileBranchId=2452987#l122)